source/index-management/index-management.rst

```diff
@@ -902,6 +933,11 @@ Common Interfaces
     name: Optional<string>;
   }
 
+  interface SearchIndexOptions {
+    // The name for this index, if present.
+    name: Optional<string>;
```

Comment:
Since the optional `name` only applies to the create operations, I suggest placing `name` in `CreateSearchIndexOptions` to simplify the API.

Reply:
I don't feel strongly here, but I intended the new search indexes API to be as similar to the existing API as possible:

```typescript
    createIndex(keys: Document, indexOptions: Optional<IndexOptions>, options: Optional<CreateIndexOptions>): String;
```

Some drivers (like PHP) do combine the options types, which has been noted in a comment on the new createSearchIndex methods (same as is done for createIndex).